src/Neo/Wallets/Wallet.cs

```diff
@@ -141,6 +141,7 @@ public WalletAccount CreateAccount()
         {
             var privateKey = new byte[32];
             using var rng = RandomNumberGenerator.Create();
+            var maxTry = 100;
```

Comment:
how much time it spend in 100 ? maybe is few number?

Reply:
this is not likly t happen,,,,,but will do 10

Reply:
Before it was infinite...eahuehauea
Any number is good to me here, but 10 looks small but unlikely to happen.